filter_plugins/oo_filters.py

```diff
@@ -881,14 +881,13 @@ def oo_image_tag_to_rpm_version(version, include_dash=False):
         """
         if not isinstance(version, basestring):
             raise errors.AnsibleFilterError("|failed expects a string or unicode")
-        # TODO: Do we need to make this actually convert v1.2.0-rc1 into 1.2.0-0.rc1
-        # We'd need to be really strict about how we build the RPM Version+Release
         if version.startswith("v"):
-            version = version.replace("v", "")
+            version = version[1:]
+            # Strip release from requested version, we no longer support this.
             version = version.split('-')[0]
```

Comment:
I'm not even sure if we are doing anything with them currently, but the tags would be the ones used for pulling containers used by openshift, for example the router and/or registry. I think they may be defaulting to using the templates defined in the master config (or the version from the client itself) now, though.